.authors.yml

```diff
@@ -1251,7 +1251,7 @@
   github: beeankha
   alternate_emails:
   - beeankha@gmail.com
-  num_commits: 43
+  num_commits: 39
```

Comment:
Not sure why the number of commits got lower here... 🤔

Reply:
I'd bump it right back up!

Reply:
The commit counts are weird for whomever last did the release due to our commit squashing policy